modules/understanding-pod-error-states.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * support/investigating-pod-issues.adoc
+
+[id="understanding-pod-error-states_{context}"]
+= Understanding Pod error states
+
+Pod failures return explicit error states which can be observed in the `status` field in the output of `oc get Pods`. Pod error states cover image, container and container network related failures.
+
+The following table provides a list of Pod error states along with their descriptions.
+
+.Pod error states
+[cols="1,4",options="header"]
+|===
+| Pod error state | Description
+
+2+^s| Images error states
```

Comment:
At first glance, I thought this line was an error. This is a little complex for the table (and might not be the best for accessibility, though I'm definitely far from an expert there). I'd suggest either removing these rows (Images error states, Container error states, Container network error states), or if you think they're important, then break up this into 3 tables.

Reply:
Thanks Andrea. I have now removed those three rows.